s3.py

```diff
@@ -14,6 +14,8 @@
 from enum import Enum
 from flask_babel import lazy_gettext as _
 
+from core.model.configuration import ConfigurationOption, ConfigurationGrouping, ConfigurationMetadata, \
+    ConfigurationAttributeType
```

Comment:
Same import problem as I mentioned before. See below for `from model import` example.